Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:14.04
+MAINTAINER Yi Wang <yi.wang.2005@gmail.com>
+
+COPY build.sh /opt
+
+CMD /opt/build.sh
```

Comment:
Sorry this explains ruin is wrong. I thought it is RUN instead of CMD. Glad it worked!